pkger/clone_resource.go

```diff
@@ -248,6 +248,8 @@ func convertCellView(cell influxdb.Cell) chart {
 	case influxdb.SingleStatViewProperties:
 		setCommon(chartKindSingleStat, p.ViewColors, p.DecimalPlaces, p.Queries)
 		setNoteFixes(p.Note, p.ShowNoteWhenEmpty, p.Prefix, p.Suffix)
+		ch.TickPrefix = p.TickPrefix
+		ch.TickSuffix = p.TickSuffix
```

Comment:
`ch` made me think these were channels at first

Reply:
Should we change the name? or you think its fine to stay?

Reply:
nah dont chagne the name please 😬, I hear you @glinton, but we all know chans don't have methods or members ;-).... or even useful zero values lol